### PR TITLE
just really minor stuff

### PR DESCRIPTION
**Pull request by alexweber**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "@zenginehq/zengine-sdk": "github:ZengineHQ/zengine-sdk",
+    "firebase": "^2.3.2",
```
Comment (tehpsalmist) [inline package.json]: I noticed that Ted was using this, so I assume this works fine with our database secrets?

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v10.15.3
```
Comment (tehpsalmist) [inline .nvmrc]: why is this file necessary?

File: package.json (hunk 1 above)
Comment (alexweber) [reply package.json]: Yep both 2.3.1 and 2.3.2 work fine, there's a bug in 2.3.1 that affected some things in the past.  Most plugins use 2.3.2 because of that.  If and when we figure out how to use the new sdk we can remove this....

File: .nvmrc (hunk 2 above)
Comment (alexweber) [reply .nvmrc]: most people have multiple versions of node installed at any given moment using nvm... when there's an `nvmrc` file you can just type in `nvm use` and it will set the expected version for you.  it's like a cue for people as to what version of node they should be using for this project.  can't hurt to be explicit about it...

Comment (tehpsalmist) [reply .nvmrc]: fair enough...

File: package.json (hunk 1 above)
Comment (tehpsalmist) [reply package.json]: oh, I didn't realize this is the version we always used.
